frontend/src/metabase/dashboard/components/DashboardHeader/buttons/ExportAsPdfButton.module.css

```diff
@@ -0,0 +1,10 @@
+.CompactExportAsPdfButton {
+  position: absolute;
+  right: 1rem;
+  top: 0.5rem;
+  z-index: 1;
+}
+
+.CompactExportAsPdfButton.ParametersVisibleWithNoTabs {
+  top: 1.75rem;
+}
```

Comment:
When in compact layout, we use absolute positioning. Also, if there are only parameters (filters) available we align it with the filter box.